src/ArtemisNetClient/Management/RequestSerializer.cs

```diff
@@ -47,6 +47,10 @@ await using (writer.ConfigureAwait(false))
                 {
                     writer.WriteNumber("auto-delete-delay", -1);
                 }
+                if (string.IsNullOrEmpty(configuration.LastValueKey) == false)
```

Comment:
We need this check as it appears that when null is sent the broker treat all messages as they would have the same key define. Effectively dropping all messages except the last one.